### ISSUE TITLE
typing "exit" from within azssh gives errors before exiting

### ISSUE THREAD

**Issue by serbrech** [noelbundick/azssh#6]
```bash
% ./azssh
Requesting a Cloud Shell.
Connecting terminal...
stephane@Azure:~$ exit
exit
panic: repeated read on failed websocket connection

goroutine 31 [running]:
github.com/gorilla/websocket.(*Conn).NextReader(0xc0003d8f20, 0x0, 0xc000078000, 0x0, 0x0, 0x0)
        /home/stephane/go/src/github.com/gorilla/websocket/conn.go:968 +0x356
github.com/gorilla/websocket.(*Conn).ReadMessage(0xc0003d8f20, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/stephane/go/src/github.com/gorilla/websocket/conn.go:1029 +0x2f
github.com/noelbundick/azssh/pkg/azssh.pumpOutput(0xc0003d8f20, 0xa5bc40, 0xc00000c018)
        /home/stephane/go/src/github.com/noelbundick/azssh/pkg/azssh/terminal.go:26 +0x6c
created by github.com/noelbundick/azssh/pkg/azssh.ConnectToWebsocket
        /home/stephane/go/src/github.com/noelbundick/azssh/pkg/azssh/terminal.go:79 +0x1c2
```

**Comment by noelbundick** [comment]
yeah... 😂 it does do that!

I’ll try to add some websocket disconnect handling and fix CTRL+C next time I hack around on this

**Comment by noelbundick** [comment]
@serbrech Fixed in [0.0.4](https://github.com/noelbundick/azssh/releases/tag/v0.0.4)
